FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -21,38 +22,42 @@
 #include <tuple>
 #include <utility>
 
+// We pin r26/r27 as PF/AF respectively, this is internal FEX ABI.
+#define REG_PF 26
```

Comment:
Using:

```cpp
constexpr auto REG_PF = ARMEmitter::Reg::r26;
constexpr auto REG_AF = ARMEmitter::Reg::r27;
```

Will allow condensing stuff like 

```cpp
str(FEXCore::ARMEmitter::Register(REG_PF).X(), STATE.R(),
    offsetof(FEXCore::Core::CpuStateFrame, State.pf_raw));
```
into

```cpp
str(REG_PF.X(), STATE.R(), offsetof(FEXCore::Core::CpuStateFrame, State.pf_raw));
```

and for cases where you need the index, you can just use its `.Idx()` member function. Also keeps it consistent with the rest of the emitter.